pkg/segment/writer/metrics/metricssegment.go

```diff
@@ -679,7 +680,7 @@ func ExtractInfluxPayload(rawCSV []byte, tags *TagsHolder) ([]byte, float64, uin
 			}
 			for index, value := range tag_set {
 				if index == 0 {
-					mName = []byte(value)
+					continue
```

Comment:
add a comment here, the first element in the tag_set is actually a dbName, which we ignore